sympy/functions/combinatorial/factorials.py

```diff
@@ -1057,3 +1057,91 @@ def _eval_is_nonnegative(self):
                 return True
             elif k.is_even is False:
                 return  False
+
+
+class multinomial(CombinatorialFunction):
+    r"""Implementation of multinomial function.
+    In mathematics, the multinomial theorem describes how to expand a power
+    of a sum in terms of powers of the terms in that sum. It is the
+    generalization of the binomial theorem from binomials to multinomials.
+
+    This is the number of ways of putting n1, n2, n3, n4....nm different objects
+    into m different boxes with nk in the kth box
+
+    Examples
+    ========
+
+    >>> from sympy import Symbol, Rational, multinomial, expand_func, symbols
```

Comment:
You have imported `expand_func`, `Rational`, but haven't used. So, I suggest to remove.

Reply:
You haven't made this change.

Reply:
I have used them. Please check.

Reply:
Okay.